opendut-lea/src/components/auth.rs

```diff
@@ -58,6 +55,19 @@ pub fn LeaAuthenticated(
                     />
                 </Transition>
             }
+
+        }
+        (Some(lea_idp_config), None) => {
+            tracing::warn!("Warning: Authentication disabled - User not authenticated.");
```

Comment:
Isn't the authentication enabled when this config object is present?

Reply:
That is right. Changed it.